storage/rocksdb/ha_rocksdb.cc

```diff
@@ -15342,10 +15342,17 @@ double ha_rocksdb::read_time(uint index, uint ranges, ha_rows rows) {
 }
 
 void ha_rocksdb::print_error(int error, myf errflag) {
-  if (error == HA_ERR_ROCKSDB_STATUS_BUSY) {
-    error = HA_ERR_LOCK_DEADLOCK;
+  switch (error) {
```

Comment:
Unfortunately, this fix spoils the situation when lock_wait_timeout occurs. It will print:
`ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded`
but should:
`ERROR 1205 (HY000): Lock wait timeout exceeded; try restarting transaction`

In other words the current behavior is (testcase from https://jira.percona.com/browse/PS-7894):
```
mysql> select /*+ MAX_EXECUTION_TIME(5) */ * from test_table_rocks where id=1 for update; 
ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded
mysql> select * from test_table_rocks where id=1 for update; 
ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded

mysql> select /*+ MAX_EXECUTION_TIME(5) */ * from test_table_inno where id=1 for update; 
ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded
mysql> select * from test_table_inno where id=1 for update; 
ERROR 1205 (HY000): Lock wait timeout exceeded; try restarting transaction
```
but should be:
```
mysql> select /*+ MAX_EXECUTION_TIME(5) */ * from test_table_rocks where id=1 for update; 
ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded
mysql> select * from test_table_rocks where id=1 for update; 
ERROR 1205 (HY000): Lock wait timeout exceeded; try restarting transaction

mysql> select /*+ MAX_EXECUTION_TIME(5) */ * from test_table_inno where id=1 for update; 
ERROR 3024 (HY000): Query execution was interrupted, maximum statement execution time exceeded
mysql> select * from test_table_inno where id=1 for update; 
ERROR 1205 (HY000): Lock wait timeout exceeded; try restarting transaction
```